addon/components/mdc-drawer.js

```diff
@@ -0,0 +1,114 @@
+import Ember from 'ember';
+import { MDCComponent } from '../mixins/mdc-component';
+import getElementProperty from '../utils/get-element-property';
+import { MDCTemporaryDrawerFoundation, MDCPersistentDrawerFoundation } from '@material/drawer';
+import layout from '../templates/components/mdc-drawer';
+
+const { Component, computed, get, getProperties, run, set } = Ember;
+
+export default Component.extend(MDCComponent, {
+//region Ember Hooks
+  layout,
+  classNames: ['mdc-drawer'],
+  classNameBindings: ['mdcClassNames', 'isPermanent:mdc-permanent-drawer'],
+
+  init() {
+    this._super(...arguments);
+    set(this, 'changeHandlers', Ember.A([]));
+  },
+
+  didInsertElement() {
+    this._super(...arguments);
+    this.updateOpenness();
+  },
+  didUpdateAttrs() {
+    this._super(...arguments);
+    this.updateOpenness();
+  },
+  //endregion
+
+  //region Attributes
+  /**
+   * @type {Boolean}
+   */
+  permanent: true,
```

Comment:
As written, if you're setting a default value to true here, shouldn't the nested components also use true?